tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_sex_gender_concepts_test.py

```diff
@@ -0,0 +1,191 @@
+"""
+Integration test for generalize_sex_gender_concepts.py module
+
+Original Issues: DC-1224
+"""
+
+# Python Imports
+import os
+from datetime import date
+
+# Third party imports
+from dateutil import parser
+
+# Project Imports
+from app_identity import PROJECT_ID
+from common import JINJA_ENV, OBSERVATION
+from cdr_cleaner.cleaning_rules.generalize_sex_gender_concepts import GeneralizeSexGenderConcepts
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+GENERALIZED_CONCEPT_ID_TEST_QUERY_TEMPLATE = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.observation`
+
+(observation_id, person_id, observation_concept_id, observation_date, observation_datetime, 
+observation_type_concept_id, value_as_number, value_as_string, value_as_concept_id, qualifier_concept_id, 
+unit_concept_id, provider_id, visit_occurrence_id, visit_detail_id, observation_source_value,
+observation_source_concept_id,unit_source_value,qualifier_source_value,value_source_concept_id,value_source_value,questionnaire_response_id)
+
+VALUES
+    (100, 1, 1585838, '2009-04-29', TIMESTAMP('2009-04-29'),
```

Comment:
Why are some of these `observation_id`(s) duplicated in the test records? (100 & 300)

Reply:
I forgot to update the test records.